enum.go

```diff
@@ -97,6 +98,25 @@ func (e Enum[M, V]) Members() []M {
 	return e.members
 }
 
+// Choice returns a randomly selected member of the enum.
+// It takes an optional seed for the random number generator.
+// nil is returned only if the Enum contains no members.
+func (e Enum[M, V]) Choice(seeds ...int64) *M {
```

Comment:
Oh no, please, don't do that for "optional" arguments. Let's do like in genesis, always accept a seed number and default to the UNIX time if it's zero.

Reply:
Sure, updated the code.